.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules
 coverage
 dist
 **/dist
+.DS_Store
```

Comment:
I was taught this:
> Platform specific git settings have a better home in your global config - that way you don't have to repeat yourself in each repo and the .gitignore file itself keeps being clean for other contributors.

so lets revert this file and add it to your global config